api/v4/source/perf.yaml

```diff
@@ -0,0 +1,75 @@
+/api/v4/perf:
```

Comment:
@hmhealey assuming `metrics`, `telemetry`, 'analytics`, must be all in the blocklist for ad-blockers. I am naively thinking we can use `perf` as the endpoint. WDYT?

Reply:
Yeah, that sounds good to me